### PR TITLE
[react-native] add measure function into ui manager

### PR DESCRIPTION
**Pull request by evollu**
it is not documented but by looking at the source code of NativeMethodsMixin you can tell these function are available
https://github.com/facebook/react-native/blob/master/Libraries/Renderer/src/renderers/native/NativeMethodsMixin.js#L72

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -7156,6 +7156,57 @@ export interface UIManagerStatic {
...
+     * are relative to the origin x, y of the ancestor view.
+     *
+     * As always, to obtain a native node handle for a component, you can use
+     * `React.findNodeHandle(component)`.
+     */
+    measureLayout(
+        node: number,
+        relativeToNativeNode: number,
+        onSuccess: MeasureLayoutOnSuccessCallback,
+        onFail: () => void /* currently unused */
```
Comment (alloy) [inline types/react-native/index.d.ts]: These callbacks are in [the wrong order](https://github.com/facebook/react-native/blob/269458c613b658a0a3e8e7e86f8394278d5b9e23/Libraries/Renderer/src/renderers/native/NativeMethodsMixin.js#L117-L118).

**Review by alloy — CHANGES_REQUESTED** [review]
While I agree these functions exist, I wonder why you’d want to expose them over the methods on the components that _are_ publicly exposed and which are already included in these types?

In any case, the lint error is legit and needs fixing as well as my one review comment.

**Comment by evollu** [comment]
I tried NativeMethodsMixin but seems like `react-native` no long exports that module.
@alloy can you verify.
I'm tried 0.42 and 0.43

**Comment by alloy** [comment]
Hmm, in retrospect I didn’t pay attention to the filename, it’s clearly a mixin and those aren’t supported on ES6 classes, so adding these is probably a good idea anyways :+1:

**Review by alloy — APPROVED** [review]

**Pull request merged**